sd.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "source": ["tokens/**/*.json"],
+  "platforms": {
+    "scss": {
+      "transformGroup": "scss",
+      "prefix": "vf",
+      "buildPath": "build/scss/",
+      "files": [
+        {
+          "destination": "_variables.scss",
```

Comment:
Let's call them `_tokens.scss` maybe? To make it clearer when we import that the file contains?